src/test/java/bio/terra/service/snapshot/SnapshotServiceTest.java

```diff
@@ -1327,19 +1329,40 @@ void makeSnapshotFromSnapshotRequestByRequestId() {
     contentsModel.requestIdSpec(requestIdModel);
     snapshotRequestModel.contents(List.of(contentsModel));
 
-    AssetSpecification asset = new AssetSpecification().name("concept_asset");
-    Dataset dataset = new Dataset().name("datasetName").assetSpecifications(List.of(asset));
+    Dataset dataset = SnapshotBuilderTestData.DATASET;
+    dataset.name("datasetName");
     when(datasetService.retrieveByName("datasetName")).thenReturn(dataset);
+    when(snapshotRequestDao.getById(snapshotAccessRequestId))
+        .thenReturn(
+            new SnapshotAccessRequestResponse()
+                .snapshotSpecification(
+                    new SnapshotBuilderRequest()
+                        .addValueSetsItem(new SnapshotBuilderFeatureValueGroup().name("Drug"))));
 
     Snapshot actual = service.makeSnapshotFromSnapshotRequest(snapshotRequestModel);
-    SnapshotSource snapshotSource = new SnapshotSource().dataset(dataset).assetSpecification(asset);
+    SnapshotSource snapshotSource = new SnapshotSource().dataset(dataset);
 
-    assertThat(actual.getTables().size(), is(0));
+    assertThat(actual.getTables().size(), is(3));
```

Comment:
When validating the size of a collection, you can use `hasSize()`
```suggestion
    assertThat(actual.getTables(), hasSize(3));
```